### DISCUSSION TITLE
Multi-band cog file is not working as expected

### DISCUSSION THREAD

**Discussion by vs-solution7** [developmentseed/titiler#784] [Q&A] [answered]
Hello, I am trying to display a multi-band (RGB colored) cog file using TiTiler, but it's not working now.

Below is an expected result and I can get it via another cog viewer [!https://www.cogeo.org/map/#/url/https%3A%2F%2Fopo-test-temp.s3.us-west-2.amazonaws.com%2Fmap-layers%2FGreenInfrastructure%2Fcontext%2Frgb%2F2023-09-01_2023-11-30_fcover_mean.tif/center/130.8563,-12.4487/zoom/14] 
![image](https://github.com/developmentseed/titiler/assets/89006370/ce30264c-b391-4c37-94f9-6bd67d58cf03)

But when I try to do it with TiTiler, I am only able to get a black screen.
![image](https://github.com/developmentseed/titiler/assets/89006370/c6691c2c-448d-4736-9a0b-3bcf0a242e04)

Please advise me on what is wrong.
Thanks

**Comment by vincentsarago** [comment, marked as answer]
Your TIF must be of datatype float which is why titiler can't display it well by default. 

```
curl https://titiler.xyz/cog/info\?url\=https%3A%2F%2Fopo-test-temp.s3.us-west-2.amazonaws.com%2Fmap-layers%2FGreenInfrastructure%2Fcontext%2Frgb%2F2023-09-01_2023-11-30_fcover_mean.tif | jq '.dtype'   

"float64"
```

You can use the `rescale` min/max entry to adjust the rescaling values 


https://titiler.xyz/cog/map?url=https%3A%2F%2Fopo-test-temp.s3.us-west-2.amazonaws.com%2Fmap-layers%2FGreenInfrastructure%2Fcontext%2Frgb%2F2023-09-01_2023-11-30_fcover_mean.tif&rescale=0,1&bidx=1&bidx=2&bidx=3&minzoom=8&maxzoom=14

**Reply by vs-solution7** [reply]
@vincentsarago  thank you for paying attention to this matter and prompt response. It works now👍
